core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseMonitorServiceImpl.kt

```diff
@@ -95,15 +108,21 @@ class DataBaseMonitorServiceImpl(
         updateParameters: QueryParametersValues,
         interval: Duration,
         updateQueryId: QueryId,
-        updateListener: UpdateListener
+        updateListener: UpdateListener,
+        messageLoader: MessageLoader
     ) {
-        val lastRow: TableRow? = dataBaseService.executeQuery(
-            dataSourceId,
-            initQueryId,
-            initParameters,
-        ).onEach {
-            updateListener.onUpdate(dataSourceId, it)
-        }.lastOrNull()
+        val properties = mapOf(
+            TH2_PULL_TASK_UPDATE_HASH_PROPERTY to dataBaseService.calculateHash(dataSourceId, updateQueryId).toString()
+        )
+
+        val lastRow: TableRow? = messageLoader.load(dataSourceId, properties)
```

Comment:
In my opinion, loading the last message from the LDP should be optional. We can make it enabled by default but in the end the user should be able to disable that functionality

Reply:
Do you mean add a option into pulling config?

Reply:
Yes, to the pulling config and to the pulling gRPC request